### PR TITLE
Use testthat v3

### PR DESCRIPTION
**Pull request by maurolepore**
The checks should pass after merging #126 (do that first).

----
This PR enables testthat v3.
https://www.tidyverse.org/blog/2020/10/testthat-3-0-0/

### CODE REVIEW

File: tests/testthat/test-get_biomass.R [hunk 1]
```diff
@@ -1,4 +1,3 @@
-context("Test all functionalities of the get_biomass function")
```
Comment (maurolepore) [inline tests/testthat/test-get_biomass.R:1]: `context()` is now deprecated.

File: tests/testthat/test-illustrate_allodb.R [hunk 2]
```diff
@@ -1,8 +1,7 @@
-context("Test the illustrate_allodb() function")
 library(allodb)
 
 test_that("illustrate_allodb() returns a ggplot object", {
-  expect_is(illustrate_allodb(genus = "Quercus",
```
Comment (maurolepore) [inline tests/testthat/test-illustrate_allodb.R:5]: `expect_is()` is now deprecated.

File: tests/testthat/test-weight_allom.R [hunk 3]
```diff
@@ -10,7 +9,7 @@ test_that("weight_allom returns a vector of the same length as the equation tabl
               coords = c(-78, 40),
               new_eqtable = newtab
             )
-            expect_is(weight_test, "numeric")
+            expect_type(weight_test, "double")
```
Comment (maurolepore) [inline tests/testthat/test-weight_allom.R:12]: "numeric" is the class; "double" is the type.

``` r
class(1)
#> [1] "numeric"
typeof(1)
#> [1] "double"
```

<sup>Created on 2021-02-13 by the [reprex package](https://reprex.tidyverse.org) (v1.0.0)</sup>

**Comment by gonzalezeb** [comment]
Not sure how to solve this conflict before merging. It seems there are extra characters in line 46 of the description file?
Maybe for Mauro to review again.

**Comment by maurolepore** [comment]
I just fixed the merge conflict. It should be good to merge.

**Pull request merged**